packages/esm-ward-app/src/createDashboardLink.component.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useMemo } from 'react';
```

Comment:
Remove this component.  If we need to add this to the home dashboard component in the future, and things are still not refactored out of the home app, we can add something like this in.  I'm hoping it will not be needed.